browser/widevine/widevine_permission_request.cc

```diff
@@ -48,9 +47,9 @@ void WidevinePermissionRequest::PermissionGranted() {
 #if BUILDFLAG(BUNDLE_WIDEVINE_CDM)
   // Run next commands at the next loop turn to prevent this is destroyed
   // by restarting process. This should be destroyed by RequestFinished().
-  PostTask(FROM_HERE,
-           { base::CurrentThread(), base::TaskPriority::BEST_EFFORT },
-           base::BindOnce(&InstallBundleOrRestartBrowser));
+  DCHECK(base::SequencedTaskRunnerHandle::IsSet());
+  base::SequencedTaskRunnerHandle::Get()->PostTask(
```

Comment:
basically it always happens on the UI thread, so DCHECK is sort of not needed. And you can just post to UI.